pymc3/distributions/__init__.py

```diff
@@ -233,8 +238,9 @@ def sample_to_measure_vars(
 def logpt(
     rv_var: TensorVariable,
     rv_value: Optional[TensorVariable] = None,
-    jacobian: Optional[bool] = True,
-    scaling: Optional[bool] = True,
+    jacobian: bool = True,
+    scaling: bool = True,
+    transformed: bool = True,
```

Comment:
Same here - inserting `, *,` would make the calling code easier to read/understand.
This function is not directly public API right?

Reply:
I do expect that `logpt` would be of interest to people implementing their own PyMC3-based code, so it's probably somewhere between public and private API.

Reply:
Okay, sounds like inserting `, *,` wouldn't break too much on the user side, but would give us more flexibility and clear code